CMakeLists.txt

```diff
@@ -6,10 +6,23 @@ project(mlboard
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+option(BUILD_TESTS "Build tests." ON)
+INCLUDE(FindProtobuf)
+FIND_PACKAGE(Protobuf REQUIRED)
 
 ADD_SUBDIRECTORY(include/proto)
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
 
+file(GLOB headerfiles "${CMAKE_CURRENT_SOURCE_DIR}/include/filewriter/*.hpp")
```

Comment:
[Globbing is controversial](https://stackoverflow.com/questions/1027247/is-it-better-to-specify-source-files-with-glob-or-each-file-individually-in-cmak/1060061#:~:text=The%20creators%20of%20CMake%20themselves%20advise%20not%20to%20use%20globbing.&text=(We%20do%20not%20recommend%20using,to%20ask%20CMake%20to%20regenerate.)), but I think it's fine here.